apps/explorer/src/components/OwnedObjects/OwnedObjectsText.tsx

```diff
@@ -0,0 +1,32 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+import { cva, type VariantProps } from 'class-variance-authority';
+import { type ReactNode } from 'react';
+
+const OwnedObjectsTextStyles = cva(
```

Comment:
what's the purpose of this additional text component?

Reply:
This is done so we can use `group-hover`. If I use the basic Text component, I'd need to use js with `onMouseOver` and `onMouseLeave` to change the text styling.